src/index.coffee

```diff
@@ -11,7 +11,7 @@ module.exports = class LiveScriptCompiler
 
   compile: (data, path, callback) ->
     try
-      result = LiveScript.compile data, bare: yes
+      result = livescript.compile data, bare: yes, header: no
```

Comment:
Ok fair enough. Didn't realize the header was something new on livescript's side.